React/Fabric/Mounting/ComponentViews/ScrollView/RCTEnhancedScrollView.mm

```diff
@@ -130,9 +130,9 @@ - (void)setDelegate:(id<UIScrollViewDelegate>)delegate
 
 #pragma mark - UIScrollViewDelegate
 
-- (void)scrollViewWillEndDragging:(RCTUIScrollView *)scrollView
+- (void)scrollViewWillEndDragging:(UIScrollView *)scrollView
```

Comment:
Is this meant to be in this PR?

Reply:
Heh.. didn't we do this in #1526 ? Some weird git thing maybe.

Reply:
Yikes, I accidentally merged meta/main instead of microsoft/main 😅. I reverted but this looks like side-effects. Will clean up!